internal/components/extensions_test.go

```diff
@@ -85,7 +85,7 @@ func TestDefaultExtensions(t *testing.T) {
 	}
 
 	// * The OIDC Auth extension requires an OIDC server to get the config from, and we don't want to spawn one here for this test.
-	assert.Equal(t, len(tests)+8 /* not tested */, len(extFactories))
```

Comment:
Please provide justification why not testing this extension.

Reply:
I've just gone ahead and added a test for it.